src/wristpy/processing/metrics.py

```diff
@@ -0,0 +1,30 @@
+"""Calculate base metrics, anglez and enmo."""
+
+import numpy as np
+
+from wristpy.core.models import Measurement
+
+
+def euclidean_norm_min_one(acceleration: Measurement) -> Measurement:
+    """Compute ENMO, the Euclidean Norm Minus One (1 standard gravity unit).
+
+    Negative values of ENMO are set to zero because ENMO is meant as a measure of
+    physival activity. Negative values steming from low levels of activity or noise from
+    the device would have no meaningful interpretation in this context and would be
+    detrimental to the intended analysis.
```

Comment:
I might change this description of the wrapping to zero slightly. It is more due to imperfect calibration + device noise. In theory, if the sensor was perfectly calibrated ENMO should never be negative (unless you were in a zero-_g_ room or in flight?). Not necessarily related to low levels of physical activity.